src/components/ResumeList/ResumeListMobile.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FC } from 'react';
+import { BoxProps } from './type';
+import { Box as MuiBox, makeStyles, Paper, Typography } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  box: {
+    backgroundColor: theme.palette.background.paper,
+  },
+
```

Comment:
It would be more consistent with the rest of the codebase if you remove newlines between object properties